library/src/test/java/io/ona/kujaku/services/MapboxOfflineDownloaderServiceTest.java

```diff
@@ -261,7 +261,7 @@ public void mapboxTileLimitExceededShouldCreateValidBroadcast() throws Interrupt
         latch.await();
 
         Intent intent = (Intent) resultsToCheck.get(0);
-        assertBroadcastResults(intent, MapboxOfflineDownloaderService.SERVICE_ACTION_RESULT.FAILED, mapName, "MapBox Tile Count limit exceeded : 60000 while Downloading " + mapName, MapboxOfflineDownloaderService.SERVICE_ACTION.DOWNLOAD_MAP);
+        assertBroadcastResults(intent, MapboxOfflineDownloaderService.SERVICE_ACTION_RESULT.FAILED, mapName, "MapBox Tile Count limit exceeded 60000 while Downloading " + mapName, MapboxOfflineDownloaderService.SERVICE_ACTION.DOWNLOAD_MAP);
```

Comment:
I think the error string can be replaced with the string resource `error_mapbox_tile_count_limit`?

Reply:
Cool

Reply:
Not sure I understand. I think it's okay for tests to fail after changes, right?

Reply:
This error string is already in `strings.xml`, if we use the string resource then the tests will not fail when this is changed in the actual application code

Reply:
okay.